docs/src/content/docs/de/guides/i18n.mdx

```diff
@@ -55,21 +57,25 @@ Starlight bietet eingebaute Unterstützung für mehrsprachige Seiten, einschlie
        - docs/
          - ar/
          - en/
-         - zh/
+         - zh-cn/
 
    </FileTree>
 
 3. Du kannst nun Inhaltsdateien in deinen Sprachverzeichnissen hinzufügen. Verwende den gleichen Dateinamen, um Seiten in verschiedenen Sprachen zuzuordnen und nutze Starlights volle i18n-Funktionen, einschließlich Fallback-Inhalte, Übersetzungshinweise und mehr.
 
    Erstelle zum Beispiel `ar/index.md` und `en/index.md`, um die Homepage für Arabisch bzw. Englisch darzustellen.
 
+</Steps>
+
+Für fortgeschrittene i18n-Szenarien unterstützt Starlight auch die Konfiguration der Internationalisierung mit der Option [Astro's `i18n` config](https://docs.astro.build/de/guides/internationalization/#i18n-routing-konfigurieren).
```

Comment:
Just to double check, is "Astro's `i18n` config" expected or something that could have been missed? Asking as it's identical to the English version.